src/fragments/lib/auth/flutter/getting_started/40_authenticator.mdx

```diff
@@ -0,0 +1,267 @@
+## Authentication with Amplify
+
+There are 2 options to add authentication capabilities to your application:
+- Option 1: Use Authenticator UI Library
+- Option 2: Call Authentication APIs manually
+
+## Option 1: Use Authenticator UI Library
+
+<Callout>
+
+Authenticator for Flutter is in developer preview.
+
+</Callout>
+
+The Flutter Authenticator library allows you to quickly create an authentication/registration flow within your app. The experience generated within your app will be determined by the `amplifyconfiguration.dart` file that is automatically generated for you after initializing your app, and adding auth or other categories. Here is an [example app](https://github.com/aws-amplify/amplify-flutter/tree/main/packages/amplify_authenticator/example) that utilizes Authenticator.
```

Comment:
Again, for clarity we might want to specify amplify-flutter.

Reply:
Addressed this now!